spec/features/publish_new_work_spec.rb

```diff
@@ -2,12 +2,12 @@
 
 require 'rails_helper'
 
-RSpec.describe 'Publishing a new work', unless: ci_build? do
+RSpec.describe 'Publishing a new work' do
   let(:user) { create(:user) }
   let(:metadata) { MetadataFactory.new.work_version }
   let(:updated_metadata) { MetadataFactory.new.work_version }
 
-  it 'routes the user through the workflow', with_user: :user, with_driver: :selenium_chrome_headless do
+  it 'routes the user through the workflow', with_user: :user, with_driver: :selenium_remote do
```

Comment:
@whereismyjetpack see @rschenk's other comment, but the option here should be that we use a local driver, and ideally have the option to specific different kinds of drivers for local debugging, etc. But when we're in a ci build, we'd have the test use `:selenium_remote`

Reply:
this is a bad name. i should mend that. i think what would happen is it would register a local driver if you don't have the `SELENIUM_URL` set. I think we should maybe wrap that all up in the `ci_build?` boolean and 🎂